app/services/game_assignments_service.rb

```diff
@@ -42,7 +42,7 @@ def retrieve_prior_game_assignments
             inning_number: inning.inning.inning_number,
             position: inning.fielding_position&.name.nil? ? "------" : inning.fielding_position.name
           }
-        end
+        end.sort_by { |ga| ga[:inning_number] }
```

Comment:
This is the same change you put in earlier. I wanted it available so I could test more easily.